kolibri/content/management/commands/content.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import os
+import shutil
+
+from builtins import input
+from django.core.management.base import BaseCommand
+
+from kolibri.utils import server
+from kolibri.utils.conf import KOLIBRI_HOME
+from kolibri.utils.conf import OPTIONS
+from kolibri.utils.options import update_options_file
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        subparsers = parser.add_subparsers(dest='command', help="The following subcommands are available.")
+        movedir_subparser = subparsers.add_parser(
+            name='movedirectory',
+            cmd=self,
+            help="Migrates the content to a specific folder defined by users."
+        )
+        movedir_subparser.add_argument(
+            'destination',
+            type=str,
+            help='Destination of the new content directory.'
+        )
+
+    def handle(self, *args, **options):
+        if options['command'] == 'movedirectory':
+            self.move_content_directory(options['destination'])
+
+    def move_content_directory(self, dst):
+        # Check exsitence of current directory before running the commands.
+        content_directory = OPTIONS['Paths']['CONTENT_DIR']
+        destination = os.path.abspath(os.path.expanduser(dst))
+
+        if not os.path.exists(content_directory):
+            self.stderr.write(
+                self.style.ERROR(
+                    '\nCurrent content directory {} does not exist.'
+                ).format(content_directory))
+            raise SystemExit()
```

Comment:
This should be `raise SystemExit(1)`, as well as with the other errors -- otherwise the command exits as if it went well, which makes it hard to call from other scripts.

Reply:
thanks! I will change them